### DISCUSSION TITLE
Keep up the good work!

### DISCUSSION THREAD

**Discussion by Homie5056** [4JX/L5P-Keyboard-RGB#65] [General]
Dude, the addition of the new ripple effect got you my full attention! Like damn man, I am well aware coding is a pain, as I am a student myself, heh. You just nailed it dude. Did something lenovo themselves couldn't. You earned my full respect and thank you for all your hardwork and hope to see more updates from you...

**Comment by 4JX** [comment]
Hehe, thanks! Progress will basically be nonexistent for a few weeks while I prepare for finals, but if you've got anything to suggest feel free to do so.
